### ISSUE TITLE
ipv6-icmp support

### ISSUE THREAD

**Issue by wsunccake** [openstack4j/openstack4j#139]
# Issue description
![ipv6-icmp](https://user-images.githubusercontent.com/5705809/111747646-77fd8d00-8887-11eb-92ce-e9c0e225dd9f.png)

Does ipv6-icmp supported by openstack version victoria?
I only find these "**TCP, UDP, ICMP, UNRECOGNIZED;**" in  https://github.com/openstack4j/openstack4j/blob/ab6f59380b26a47a366931ef803e13a70df4b6fc/core/src/main/java/org/openstack4j/model/compute/IPProtocol.java
...

## Submitter checklist

Make sure that following is specified to make the issue easier to process:

- [ ] The issue description links to OpenStack API documentation of that particular endpoint/request (https://docs.openstack.org/api-quick-start/#current-api-versions).
- [ ] OpenStack service name is used as an issue name prefix (ex.: "Neutron: Add floatingIp port forwardings service").
- [ ] If applicable, add GitHub link to classes that needs extending.

**Comment by olivergondza** [comment]
There seems to be lot more protocols available in openstack: https://docs.openstack.org/api-ref/network/v2/?expanded=list-security-groups-detail,list-security-group-rules-detail#list-security-group-rules

> The IP protocol can be represented by a string, an integer, or null. Valid string or integer values are any or 0, ah or 51, dccp or 33, egp or 8, esp or 50, gre or 47, icmp or 1, icmpv6 or 58, igmp or 2, ipip or 4, ipv6-encap or 41, ipv6-frag or 44, ipv6-icmp or 58, ipv6-nonxt or 59, ipv6-opts or 60, ipv6-route or 43, ospf or 89, pgm or 113, rsvp or 46, sctp or 132, tcp or 6, udp or 17, udplite or 136, vrrp or 112. Additionally, any integer value between [0-255] is also valid. The string any (or integer 0) means all IP protocols. See the constants in neutron_lib.constants for the most up-to-date list of supported strings.

**Comment by wsunccake** [comment]
I know these protocols in openstack, but i can't find these in openstack4j.
Will new openstack4j support these protocol?

**Comment by olivergondza** [comment]
Agreed, those are not yet supported by openstack4j, and they should be. I attached the details for completeness.

**Comment by bboyHan** [comment]
we'll try to solve it.
